app/controllers/api/v0/search_controller.rb

```diff
@@ -1,47 +1,64 @@
-class Api::V0::SearchController < ApplicationController
+class Api::V0::SearchController < Api::V0::BaseController
 
-  def search
-    # TODO generalize
-    query = {
-      "size": 25,
-      "query": {
-        "multi_match": {
-          "query": params['q']
-          # "fields": ["fields.title^4", "fields.plot^2", "fields.actors", "fields.directors"]
-        }
-      },
-      "_source": ["id"],
-       "highlight": {
-          "fields": {
-            "content": {}
-          }
-        }
-    }
-
-    response = OpenSearch::ElasticsearchClient.instance.search body: query.to_json
-    render json: response
+  swagger_path '/search' do
```

Comment:
so this path is going to end up looking something like `openstax.org/search/api/v0/search` right?

Reply:
Yes... `openstax.org/open-search/api/v0/search`.  The `/api/v0` part is defined as an API base somewhere else... but this is a good point that we don't have the `open-search` prefix there.  We could add it and just make all Swagger use assume access via CloudFront (and that prefix) or we could have two Swagger JSONs, one with and one without the prefix.

Reply:
we can just use the prefix all the time right? no reason to support queries without the prefix like accounts does

Reply:
I updated to use the `/open-search` prefix in the swagger json